src/page/product-added-modal.page.ts

```diff
@@ -7,7 +7,7 @@ export class ProductAddedModalPage {
     this.modalButton = $('[style*="display: block;"] .button-container > a');
   }
 
-  public async addedModal(): Promise<void> {
+  public async clickModalButton(): Promise<void> {
```

Comment:
in fact, we are clicking on proceed to checkout button, since there are two buttons on that modal page, the name that you are using can be confusing.